src/main/kotlin/no/nav/helse/spion/auth/altinn/AltinnTypes.kt

```diff
@@ -0,0 +1,13 @@
+package no.nav.helse.spion.auth.altinn
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+data class AltinnOrganisasjon(
+        @JsonProperty("Name") val name : String,
```

Comment:
hmm, dette er for å få tilbake responsen fra altinn? Det virker veldig redundant da, kan man ikke eventuelt konfigurere deserializeren til å være case insensitive?

Reply:
Hmm, ja, kanskje det. Jeg tenkte egentlig å dele så mye som mulig på JSONSerialiserer-oppsettet (bruke samme ObjectMapper instans og den gjennom Koin) slik at vi ikke ender opp med 15 forskjellige de/serialiserere som er confet forskjellig og ikke kan snakke med hverandre, men det poenget er kanskje mute ifbm integrasjonene uansett i og med at vi da er prisgitt formatet vi får fra 3.part.

Her er det ganske teit at jeg må gjøre dette 2 ganger, en gang i prod-koden og en gang i test-koden. Skal prøve å gjøre dette litt mer sleek.

Reply:
La inn en case-in-sensitive objectmapper for bruk i integrasjoner 👍